lib/gem2rpm/package.rb

```diff
@@ -1,4 +1,6 @@
 require 'delegate'
+# require 'rubygems' to avoid NameError at Ruby 1.8.7.
+require 'rubygems' unless defined?(Gem)
```

Comment:
Okay. I could check that real case passed without "require 'rubygems'" on RHEL 6.5, with ruby 1.8.7.

```
$ ruby -v
ruby 1.8.7 (2013-06-27 patchlevel 374) [x86_64-linux]

$ gem list

*** LOCAL GEMS ***

minitest (1.6.0)
rake (0.8.7)

$ bin/gem2rpm -T
...
- Initial package

$ echo $?
0

$ ruby lib/gem2rpm/package.rb

$ echo $?
0
```

However I am still confused.
Does its NameError happen only in the case of using minitest on ruby 1.8.7?
If it happens to that case, do we need to modify Rakefile or test files to pass the test (rake test) with ruby 1.8.7 on Travis? Because above error stack trace is for Travis ruby 1.8.7 case.